pkgs/applications/video/giph/default.nix

```diff
@@ -0,0 +1,63 @@
+{ stdenv
+, lib
+, fetchFromGitHub
+, ffmpeg
+, xdotool
+, withSlop ? true
+, slop
+, withNotify ? true
+, libnotify
+, withPgrep ? true
+, procps
+}:
+
+stdenv.mkDerivation rec {
+  pname = "giph";
+  version = "1.1.1";
+
+  src = fetchFromGitHub {
+    owner = "phisch";
+    repo = "giph";
+    rev = version;
+    sha256 = "19l46m1f32b3bagzrhaqsfnl5n3wbrmg3sdy6fdss4y1yf6nqayk";
+  };
+
+  postPatch = ''
+    substituteInPlace src/giph \
+      --replace '=(ffmpeg' '=(${ffmpeg}/bin/ffmpeg' \
+      --replace '="$(xdotool' '="$(${xdotool}/bin/xdotool' \
+  '' + lib.optionalString withSlop ''
+      --replace '=(ffmpeg' '=(${ffmpeg}/bin/ffmpeg' \
```

Comment:
this line doesn't look correct, why does it replace ffmpeg when `withSlop` is checked?